web/src/inbox_ui.ts

```diff
@@ -584,7 +584,9 @@ function format_stream(stream_id: number): StreamContext {
         is_archived: stream_info.is_archived,
         invite_only: stream_info.invite_only,
         is_web_public: stream_info.is_web_public,
-        stream_name: stream_info.name,
+        stream_name: stream_info.is_archived
+            ? `${stream_info.name} (archived)`
+            : stream_info.name,
```

Comment:
This looks to be accidentally included but part of something else you were working on?